common/lib/xmodule/xmodule/js/spec/capa/display_spec.coffee

```diff
@@ -125,6 +125,19 @@ describe 'Problem', ->
       it 're-bind the content', ->
         expect(@problem.bind).toHaveBeenCalled()
 
+    describe 'with timed exam', ->
+      beforeEach ->
+        @problem.el.append '''
+          <div class="problem-timer" data-seconds-left="120" data-minutes-before-warning="1">
+            You have <span class="minutes-left"></span> minutes left to complete these questions.
+          </div>
+        '''
+        spyOn @problem, 'maybeSetupTimer'
+        @problem.render "Hello world"
```

Comment:
Most of this new code uses parens for function calls. Despite being optional in CS, I'd like to opt for consistency and use them here and throughout.